aeneas/src/core/Operator.v3

```diff
@@ -339,6 +339,7 @@ component V3Op {
 			V3Kind.CLASS,
 			V3Kind.FUNCREF => opcode = Opcode.RefEq;
 			V3Kind.VARIANT => opcode = Opcode.VariantEq;
+			V3Kind.REF => opcode = Opcode.RefEq;
```

Comment:
Sorry, this is a bit confusing here, but the `RefEq` operator works on "pointer" references, which are a single word. It's probably fine to leave this as an overloaded equal for now, because it will become a comparison of the byte array and offset in the middle of the compiler (similar to how ranges work).